frontend/src/views/Collection/index.tsx

```diff
@@ -34,39 +28,26 @@ import {
   LinkContainer,
   StyledDiv,
 } from "./style";
+import {
```

Comment:
extracted a few help functions into the util folder, and the remove imports above got moved into the util folder too!